beeai/agents/constants.py

```diff
@@ -12,3 +12,27 @@
 
 I_AM_JOTNAR = "by Jotnar, a Red Hat Enterprise Linux packaging AI agent."
 CAREFULLY_REVIEW_CHANGES = "Carefully review the changes and make sure they are correct."
+
+class JiraLabels:
+    """Constants for Jira labels used by Jotnar agents"""
+    REBASE_IN_PROGRESS = "jotnar_rebase_in_progress"
+    BACKPORT_IN_PROGRESS = "jotnar_backport_in_progress"
+    NEEDS_ATTENTION = "jotnar_needs_attention"
+    NO_ACTION_NEEDED = "jotnar_no_action_needed"
+
+    REBASED = "jotnar_rebased"
+    BACKPORTED = "jotnar_backported"
+
+    REBASE_ERRORED = "jotnar_rebase_errored"
+    BACKPORT_ERRORED = "jotnar_backport_errored"
+    TRIAGE_ERRORED = "jotnar_triage_errored"
+
+    REBASE_FAILED = "jotnar_rebase_failed"
+    BACKPORT_FAILED = "jotnar_backport_failed"
+
+    RETRY_NEEDED = "jotnar_retry_needed"
+
+    @classmethod
+    def all_labels(cls) -> set[str]:
+        """Return all Jotnar labels for cleanup operations"""
+        return {v for v in vars(cls).values() if isinstance(v, str)}
```

Comment:
Wouldn't an enum be better here? You can get all values easily: `l.value for l in cls`